frontend/src/components/layouts/MainLayout.tsx

```diff
@@ -1,21 +1,34 @@
 import { Layout } from 'antd';
-
-import { MenuItem, Icon } from '../navigation';
-import { Header, Content } from './components';
-
-const { Sider } = Layout;
+import { ProfileFilled } from '@ant-design/icons';
+import { MenuItem } from '../navigation';
+import { Header, Content, Sider} from './components';
+import { Breadcrumb } from 'antd';
 
 const MainLayout: React.FC = ({ children }) => (
   <Layout style={{ minHeight: '100vh' }}>
-    <Sider width={260}>
-      <MenuItem>
-        <Icon />
-        Invoice
-      </MenuItem>
+    <Sider>
+      <MenuItem title="Invoice" url="Invoice" icon ={<ProfileFilled/>}></MenuItem>    
```

Comment:
Zróbmy trochę inaczej, sama definicja modelu jest już ok. Dodaj 2 `MenuItem`, które będą przenosić odpowiednio na strony:

1. Użytkownicy -> zakładka `/users` (do tego ikona użytkowników)
2. Komponenty -> strona `/utils` (to tego ikona jakiegoś koła zębatego)

I dodatkowo nie dodawaj spacji po `icon`. Czyli zamiast `icon =` powinno być `icon=`